### PR TITLE
Add tag importer progress bar and some additional worker behaviour

### PR DESCRIPTION
**Pull request by mobaig**
See commits for details.

### CODE REVIEW

File: app/assets/javascripts/import-progress.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+$(function updateImportProgress() {
```
Comment (tijmenb) [inline app/assets/javascripts/import-progress.js]: We can use the framework in govuk_admin_template for this, for consistency with other publishing apps:

https://github.com/alphagov/govuk_admin_template/blob/master/JAVASCRIPT.md

Comment (mobaig) [inline app/assets/javascripts/import-progress.js]: Hmm...coulda sworn I tried this and found it to be a bit broken. Decided to drop it on the basis that the function is pretty simple and I couldn't find a working example in collections-publisher or service manual. Should have checked the publisher app!

Anyway, second time's a charm.

**Comment by tijmenb** [comment]
🎉 very cool!

**Comment by tijmenb** [comment]
BTW we should revisit this when we've upgraded to Rails 5. Super curious if ActionCable is easy enough to be used for this.

**Pull request merged**